src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs

```diff
@@ -26,11 +26,12 @@ internal class OpenTelemetryProtocolExporterEventSource : EventSource
         public static readonly OpenTelemetryProtocolExporterEventSource Log = new OpenTelemetryProtocolExporterEventSource();
 
         [NonEvent]
-        public void FailedToReachCollector(Exception ex)
+        public void FailedToReachCollector(Uri collectorUri, Exception ex)
```

Comment:
Didn't went with backwards compatibility since it's still in beta

Reply:
OTLP Exporter is already stable (1.1.0 is the latest stable).
But this is not public API, so not relevant.